src/test/java/teammates/test/cases/action/InstructorFeedbackResultsDownloadActionTest.java

```diff
@@ -163,6 +176,33 @@ public void testExecuteAndPostProcess() throws Exception {
         expectedFileName = session.getCourseId() + "_" + session.getFeedbackSessionName();
         assertEquals(expectedFileName, result.getFileName());
         verifyFileContentForDownloadWithMissingResponsesHidden(result.getFileContent(), session);
+
+        ______TS("Typical case: question results downloadable");
```

Comment:
question results do not suggest results of one question to me. something like : 
1. results for only one question downloadable
2. results downloadable by question